### ISSUE TITLE
MacOS AccessDenied

### ISSUE THREAD

**Issue by Scitator** [minerllabs/minerl#96]
installed with:
```
brew tap AdoptOpenJDK/openjdk
brew cask install adoptopenjdk8

conda create -n minerl python=3.7 anaconda
source activate minerl

pip install --upgrade minerl
```

code:
```python
import gym
import minerl

env = gym.make('MineRLNavigateDense-v0')
```

output
```
---------------------------------------------------------------------------
PermissionError                           Traceback (most recent call last)
~/anaconda3/envs/minerl/lib/python3.7/site-packages/psutil/_psosx.py in wrapper(self, *args, **kwargs)
    338         try:
--> 339             return fun(self, *args, **kwargs)
    340         except OSError as err:

~/anaconda3/envs/minerl/lib/python3.7/site-packages/psutil/_psosx.py in connections(self, kind)
    527         with catch_zombie(self):
--> 528             rawlist = cext.proc_connections(self.pid, families, types)
    529         ret = []

PermissionError: [Errno 1] Operation not permitted

During handling of the above exception, another exception occurred:

AccessDenied                              Traceback (most recent call last)
<ipython-input-2-b35ad76686a2> in <module>
----> 1 env = gym.make('MineRLNavigateDense-v0')

~/anaconda3/envs/minerl/lib/python3.7/site-packages/gym/envs/registration.py in make(id, **kwargs)
    154 
    155 def make(id, **kwargs):
--> 156     return registry.make(id, **kwargs)
    157 
    158 def spec(id):

~/anaconda3/envs/minerl/lib/python3.7/site-packages/gym/envs/registration.py in make(self, path, **kwargs)
     99             logger.info('Making new env: %s', path)
    100         spec = self.spec(path)
--> 101         env = spec.make(**kwargs)
    102         # We used to have people override _reset/_step rather than
    103         # reset/step. Set _gym_disable_underscore_compat = True on

~/anaconda3/envs/minerl/lib/python3.7/site-packages/gym/envs/registration.py in make(self, **kwargs)
     71         else:
     72             cls = load(self._entry_point)
---> 73             env = cls(**_kwargs)
     74 
     75         # Make the enviroment aware of which spec it came from.

~/anaconda3/envs/minerl/lib/python3.7/site-packages/minerl/env/core.py in __init__(self, xml, observation_space, action_space, port, noop_action, docstr)
    129         self._already_closed = False
    130 
--> 131         self.init(observation_space, action_space, port=port)
    132 
    133     def init(self,  observation_space, action_space,  port=None):

~/anaconda3/envs/minerl/lib/python3.7/site-packages/minerl/env/core.py in init(self, observation_space, action_space, port)
    153                 self.instance = InstanceManager.add_existing_instance(port)
    154             else:
--> 155                 self.instance = InstanceManager.get_instance()
    156         # Parse XML file
    157         with open(self.xml_file, 'r') as f:

~/anaconda3/envs/minerl/lib/python3.7/site-packages/minerl/env/malmo.py in get_instance(cls)
     98             if len(cls._instance_pool) < cls.MAXINSTANCES:
     99                 cls.ninstances += 1
--> 100                 inst = cls.Instance(cls._get_valid_port())
    101                 cls._instance_pool.append(inst)
    102                 inst._acquire_lock()

~/anaconda3/envs/minerl/lib/python3.7/site-packages/minerl/env/malmo.py in _get_valid_port(cls)
    158     def _get_valid_port(cls):
    159         port = (cls.ninstances  % 5000) + 9000
--> 160         while cls._is_port_taken(port) or \
    161               cls._is_display_port_taken(port - 9000, cls.X11_DIR) or \
    162               cls._port_in_instance_pool(port):

~/anaconda3/envs/minerl/lib/python3.7/site-packages/minerl/env/malmo.py in _is_port_taken(port, address)
    142     @staticmethod
    143     def _is_port_taken(port, address='0.0.0.0'):
--> 144         ports = [x.laddr.port for x  in psutil.net_connections()]
    145         return port in ports
    146 

~/anaconda3/envs/minerl/lib/python3.7/site-packages/psutil/__init__.py in net_connections(kind)
   2261     On macOS this function requires root privileges.
   2262     """
-> 2263     return _psplatform.net_connections(kind)
   2264 
   2265 

~/anaconda3/envs/minerl/lib/python3.7/site-packages/psutil/_psosx.py in net_connections(kind)
    250     for pid in pids():
    251         try:
--> 252             cons = Process(pid).connections(kind)
    253         except NoSuchProcess:
    254             continue

~/anaconda3/envs/minerl/lib/python3.7/site-packages/psutil/_psosx.py in wrapper(self, *args, **kwargs)
    342                 raise NoSuchProcess(self.pid, self._name)
    343             if err.errno in (errno.EPERM, errno.EACCES):
--> 344                 raise AccessDenied(self.pid, self._name)
    345             raise
    346         except cext.ZombieProcessError:

AccessDenied: psutil.AccessDenied (pid=48756)

```

**Comment by MadcowD** [comment]
This was introduced by a new change to use PSUTIL. On mac I guess you need admin for this call, I'll fix in next update.

For now can you run your python script with sudo>

**Comment by MadcowD** [comment]
@Scitator This was fixed! Make sure to upgrade to `minerl` v`0.1.15`!
```
pip3 install --upgrade minerl
```
